en/docs/references/concepts/authorization/jarm.md

```diff
@@ -0,0 +1,177 @@
+            Possible values:
+            <ul>
+                <li>
+                    <code>code</code>: Used if the <a href="../authorization-code-grant">authorization code flow</a> is implemented.
+                </li>
+                <li>
+                    <code>token</code>: Used if the <a href="../implicit-grant">implicit flow</a> is implemented.
+                </li>
+                <li>
+                    <code>code id_token</code>: Used if the <a href="../openid-connect-hybrid-flow">hybrid flow</a> is implemented.
+                </li>
+            </ul>
+        </td>
+    </tr>
+    <tr>
+        <th>
+            <code>response_mode</code>
+        </th>
+        <td>
+            Specifies how the authorization information should be returned to the client. By default, the following response modes are supported in WSO2 Identity Server:
+            <ul>
+                <li><code>query</code></li>
+                <li><code>fragment</code></li>
+                <li><code>form_post</code></li>
+            </ul>
+            In the default authorization flow (for default response modes), the authorization response parameters, such as the authorization code and ID token, are sent as human-readable plain text to the redirect URL, as shown below.</br></br>
+            <code>https://&lt;CLIENT_HOST&gt;/redirects/redirect1?code=&lt;AUTH_CODE&gt;&session_state=&lt;SESSION_STATE&gt;</code></br></br>
+            The JARM specification is used to send this response more securely by encoding the response parameters in a JWT.
+        </td>
+    </tr>
+</table>
+
+## Enable JARM
+
+By default, JARM response modes are not enabled in WSO2 Identity Server. 
+
+To enable JARM, add the following configurations to the `deployment.toml` file of the WSO2 Identity Server:
+
+```toml
+[oauth.jarm]
+jarm_response_jwt_validity = 7200
+
+[[oauth.response_mode]]
+name = "jwt"
+class = "org.wso2.carbon.identity.oauth2.responsemode.provider.jarm.impl.JwtResponseModeProvider"
+
+[[oauth.response_mode]]
+name = "query.jwt"
+class = "org.wso2.carbon.identity.oauth2.responsemode.provider.jarm.impl.QueryJwtResponseModeProvider"
+
+[[oauth.response_mode]]
+name = "fragment.jwt"
+class = "org.wso2.carbon.identity.oauth2.responsemode.provider.jarm.impl.FragmentJwtResponseModeProvider"
+
+[[oauth.response_mode]]
+name = "form_post.jwt"
+class = "org.wso2.carbon.identity.oauth2.responsemode.provider.jarm.impl.FormPostJwtResponseModeProvider"
+```
+
+## Using JARM
+
+When [JARM is enabled](#enable-jarm), clients can request authorization response parameters along with additional data in JWT format instead of plain text.
+
+The specification defines the following response modes, which you can specify using the `response_mode` parameter in the authorization request.
+
+- [response_mode=query.jwt](#queryjwt)
+- [response_mode=fragment.jwt](#fragmentjwt)
+- [response_mode=form_post.jwt](#form_postjwt)
+- [response_mode=jwt](#jwt)
```

Comment:
Added this as a new section, is it correct to add the response format like this?

Reply:
yeah +1